ui/packages/catalog-ui-search/src/main/webapp/component/result-form-selector/result-form-selector.hbs

```diff
@@ -12,5 +12,5 @@
  **/
  --}}
 
-<div class="title">Result Form Browser</div>
+<div class="title">Data View Browser</div>
```

Comment:
Did we agree on data view or result form?

Reply:
Result Form